WordPress/gutenberg
# [RNMobile] Remove opacity change when images are being uploaded

## What?
Removes opacity change when Images are uploaded.

## Why?
As part of ongoing efforts to improve the media upload experience, we have received feedback that Image upload process could be improved by modifying an "in progress" state when uploading Images.

## How?
In the current production app, an Image will appear at 0.3 opacity while uploading, and transitions to 100% visibility when the upload is successful. 

This PR removes the animation and associated code references when Images transition from local media to network media.


## Testing Instructions
1. Open a post or page and add an Image
2. Observe that the Image is full visibility while uploading (the horizontal progress bar should also be visible at the top of the Image).

## Screenshots or screencast 
<details>
<summary>Example Video</summary>

<video src="https://github.com/WordPress/gutenberg/assets/643285/bcfb7144-ebf0-4941-8f62-0584ee7aae58" />

</details>
